packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -237,6 +236,16 @@
     }
 }
 
+.cfd-jurisdiction-card--financial {
+    &__footnotes-container {
+        @include desktop {
+            gap: 1rem;
+            min-height: 7.6rem;
```

Comment:
this will cause overflow and scrolling on financial because of having longer content.... it is possible to use conditional heights, i.e. 69rem for derived and swap free, 76rem for financial but it might become a best practice issue as mentioned in above review

cc: @mahdiyeh-deriv